src/Reddit/Reddit.ts

```diff
@@ -1,427 +1,644 @@
-import { ExecutionContext } from "ava";
+import { ExecutionContext } from 'ava'
+import { setStyles, selectAll, select } from '../helpers'
 
 /*
  * LipSurf plugin for Reddit.com
  */
 /// <reference types="@lipsurf/types/extension"/>
-declare const PluginBase: IPluginBase;
+declare const PluginBase: IPluginBase
```

Comment:
you need to follow the style of the rest of the repo. We use semicolons at the end of lines.